agent/README.md

```diff
@@ -17,7 +17,7 @@ Monitor Agent 是使用 `Python` 作为脚本语言，使用 `cita-cli` 工具
 ```
 cd agent
 cp .env.example .env
-修改 .env 配置文件中 HOSTNAME、NODE_IP、NODE_PORT、NODE_DIR、SOFT_PATH、CITA_NODENAME、CITA_CHAIN_ID、CITA_NETWORKPORT 为实际部署的信息
```

Comment:
why removing `CITA_CHAIN_ID`? it is stilling using in `.env.example` as `CITA_CHAIN_ID=opc-devnet/3`

Reply:
you're removing env variable, this is a breaking change, should update the commit message

Reply:
These variables have not been used actually in previous versions. I revert changes temporarily，and edit these variables in next change.